doc/yagpdbcc_templates.txt

```diff
@@ -0,0 +1,359 @@
+channel object.
+
+Field                   Description
+-----------------------------------------------------------------------------
+.Bitrate .............. Bitrate. Only set on voice channels.
+.GuildID .............. Guild ID of the channel.
+.ID ................... The ID of the channel.
+.IsPrivate ............ Whether this channel is private.
+.IsThread ............. Whether this channel is a thread.
+.Mention .............. Mentions the channel, like #general.
+.Name ................. The channel's name.
+.NSFW ................. Whether this channel is marked as NSFW.
+.ParentID ............. The ID of this channel's parent (i.e. category).
+                        Will be 0 if no parent category.
+.Position ............. Channel's position, starting at the top.
+.Topic ................ Channel's topic.
+.Type ................. The type of this channel.
+
+
+Function                Description
+-----------------------------------------------------------------------------
+editChannelName         Edit channel's name.
+    channel             Can be ID, "name", or nil.
+    "new name" ........ Type string.
+
+editChannelTopic        Edit channel's topic.
+    channel             Can be ID, "name", or nil.
+    "new topic" ....... Type string.
+
+getChannel
+    channel ...........
+
+                           https://docs.yagpdb.xyz/reference/templates#channel
+
+
+DBEntry~
+                                                                 *yagpdbcc-db*
+-----------------------------------------------------------------------------
+All entries in this section are to be preceded with a valid DBEntry object, as
+obtained via `dbGet`.
+
+Field                   Description
+-----------------------------------------------------------------------------
+.ID ................... The ID of this entry.
+.GuildID .............. ID of the server.
+.UserID ............... Associated user ID this entry is stored under.
+.User ................. User object, will be invalid with invalid user ID.
+.CreatedAt ............ When this entry was created.
+.UpdatedAt ............ When this entry was last updated.
+.ExpiresAt ............ When this entry will expire.
+.Key .................. The key of this entry.
+.Value ................ The value of this entry.
+
+                           https://docs.yagpdb.xyz/reference/templates#dbentry
+
+
+Guild~
+                                                               *yagpdbcc-guild*
+-----------------------------------------------------------------------------
+All entries in this section are to be preceded with `.Guild`.
+
+Field                   Description
+-----------------------------------------------------------------------------
+.AfkChannelID ......... Returns the AFK Channel ID.
+.AfkTimeout ........... Time after which a user is moved into the AFK Channel.
+.Channels ............. Slice of |yagpdbcc-channel| objects.
+.EmbedEnabled ......... DEPRECATED whether guild widget is enabled.
```

Comment:
Though this is marked as deprecated and we'd have any reason to **not** include this, it's still in the official documentation.
Thus I currently believe we should keep it as well, as I think the documentation files should be an (almost) exact copy of the online variant.

Reply:
The corresponding entry in the official documentation has been removed 🧹

Reply:
Thank you Joe!
I'll remove this one (and any other as well) then.

Reply:
Addressed in 1665061